cogs/meme.py

```diff
@@ -45,6 +45,29 @@ def c_to_k(self, c):
                        f" User is now {celsius}°C "
                        f"({fahrenheit}°F, {kelvin}K).")
 
+    @commands.check(check_if_staff_or_ot)
+    @commands.command(hidden=True, name="boil")
+    async def warm_member(self, ctx, user: discord.Member):
+        """Boils a user >:E"""
+        celsius = 100
```

Comment:
Why not use something random? And if you're going to hardcode it, why not hardcode the values too?